org.eclipse.xtext.ide.tests/testlang-src/org/eclipse/xtext/ide/tests/testlanguage/ide/server/CodeActionService.xtend

```diff
@@ -46,7 +47,7 @@ class CodeActionService implements ICodeActionService {
 				case UNSORTED_MEMBERS: commands += d.fixUnsortedMembers(document, resource, params)
 			}
 		}
-		return commands
+		return commands.map[Either.forLeft(it)];
```

Comment:
this is just to make it compile right? so we dont consider to support the new api fully?

Reply:
See: #821.